SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2039,16 +2040,36 @@ void USpatialNetDriver::TickDispatch(float DeltaTime)
 				ClientConnectionManager->Advance();
 			}
 
+			if (IsValid(PlayerSpawner))
```

Comment:
Is there any method to the ordering of these systems advancing? If so, please add a comment. Otherwise I'd add them together at the end.

Reply:
I've put these https://github.com/spatialos/UnrealGDK/pull/2941/files/034a7c457238b899031b62f928e3970b54537639#diff-1fd513a8825444aa62fa45e37278bb7af7b4d043c6be10427e2a13968f8176ffR2063-R2072 here since they are debug-related so grouped with SpatialDebuggerSystem; others, I've put wherever.
I've grouped all of them together, good suggestion.